src/sort-filter-bar/sort-filter-bar.ts

```diff
@@ -216,28 +216,37 @@ export class SortFilterBar
   private disconnectResizeObserver(
     resizeObserver: SharedResizeObserverInterface
   ) {
-    resizeObserver.removeObserver({
-      target: this.sortSelectorContainer,
-      handler: this,
-    });
+    if (this.sortSelectorContainer) {
```

Comment:
I have covered this part in one of my earlier PR https://github.com/internetarchive/iaux-collection-browser/pull/329. we needs to rebase @latonv original branch for this fix.